src/tests/multihost/alltests/test_services.py

```diff
@@ -153,3 +159,42 @@ def test_0005_sssd_stops_monitoring(self, multihost):
         group_restore = "cp -vf /etc/group_bkp /etc/group"
         multihost.client[0].run_command(group_restore, raiseonerr=False)
         assert status == "PASS"
+
+    @pytest.mark.tier1_2
+    def test_0006_bz1909755(self, multihost, backupsssdconf):
+        """
+        :title: Suppress log message "[sssd] [service_signal_done]
+         (0x0010): Unable to signal service [2]:
+         No such file or directory" during logrote
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1909755
+        :id: a4f5d404-070b-11ec-8055-845cf3eff344
+        :steps:
+          1. Find main sssd process id
+          2. Send SIGHUP
+          3. There should not be any logs for
+          Unable to signal service .* No such
+          file or directory
+          modifyTimestamp in the filter
```

Comment:
The above 3 lines should be indented to point 3

Reply:
done